src/gpio/gpio.c

```diff
@@ -44,13 +45,13 @@ static mraa_result_t
 mraa_gpio_get_valfp(mraa_gpio_context dev)
 {
     char bu[MAX_SIZE];
-    sprintf(bu, SYSFS_CLASS_GPIO "/gpio%d/value", dev->pin);
+
+    sprintf(bu, SYSFS_CLASS_GPIO "%s/value", dev->gpio_path);
```

Comment:
IMHO the original spacing was more natural as declaring `bu` and populating it were grouped together. Now populating is grouped with `value_fp`-related code, which is unrelated to `bu`